pkg/config/config.go

```diff
@@ -1388,8 +1388,10 @@ func findUnknownEnvVars(config Config, environ []string) []string {
 		"DD_SERVICE":                   {},
 		"DD_DOTNET_TRACER_HOME":        {},
 		"DD_SERVERLESS_APPSEC_ENABLED": {},
-		// this variable is used by CWS functional tests
-		"DD_TESTS_RUNTIME_COMPILED": {},
+		// these variables are used by CWS functional tests
+		"DD_RUNTIME_SECURITY_CONFIG_RUNTIME_COMPILATION_ENABLED": {},
+		"DD_SYSTEM_PROBE_BPF_DIR":                                {},
```

Comment:
Couldn't this list potentially include all system-probe env vars?

Reply:
Yes it might, I'm mainly trying to clean up the output of our tests for now, but yes it could

Reply:
Do you know if it's possible to query all possible env variables used by the system probe ? Maybe this warrants another PR if the logic gets more complicated ?

Reply:
Maybe we need a boolean passed into `config.LoadCustom` to not warn of unknown env vars when loading the datadog config from system-probe?

Reply:
`sysprobeconfig.GetEnvVars()` would give you what you want.

Reply:
Ah yes that would be a better idea, let me do another PR implementing this